emission/net/usercache/builtin_usercache_handler.py

```diff
@@ -160,6 +162,21 @@ def storeViewsToCache(self):
         valid_key_list = ["diary/trips-%s"%day for day in day_list_bins.iterkeys()]
         self.delete_obsolete_entries(uc, valid_key_list)
 
+    def storeCommonTripsToCache(self):
+        """ 
+        Determine which set of common trips to send to the usercache. 
+        As of now we will run the pipeline on the full set of data and send that up
+        """
+        tour_model = esdtmpq.get_tour_model(self.user_id)
+        uc = enua.UserCache.getUserCache(self.user_id)
+        logging.debug("Adding common trips for user_id %s" % self.user_id)
+        uc.putDocument("common_trips-%s" % tour_model.user_id, tour_model)
```

Comment:
Okay

On Mon, Mar 7, 2016 at 11:32 PM, shankari <notifications@github.com> wrote:

> In emission/net/usercache/builtin_usercache_handler.py
> <https://github.com/e-mission/e-mission-server/pull/238#discussion_r55324897>
> :
>
> > @@ -160,6 +162,21 @@ def storeViewsToCache(self):
> >          valid_key_list = ["diary/trips-%s"%day for day in day_list_bins.iterkeys()]
> >          self.delete_obsolete_entries(uc, valid_key_list)
> >
> > +    def storeCommonTripsToCache(self):
> > +        """
> > +        Determine which set of common trips to send to the usercache.
> > +        As of now we will run the pipeline on the full set of data and send that up
> > +        """
> > +        tour_model = esdtmpq.get_tour_model(self.user_id)
> > +        uc = enua.UserCache.getUserCache(self.user_id)
> > +        logging.debug("Adding common trips for user_id %s" % self.user_id)
> > +        uc.putDocument("common_trips-%s" % tour_model.user_id, tour_model)
>
> Ah, I see that the key just includes the user_id, the second parameter is
> the object that you are inserting. But you do have to remove the user_id
> because as I said, it is not available on the phone.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/238/files#r55324897>.
>